docs/api.md

```diff
@@ -131,8 +131,14 @@ Prints the version of your `nativefier` install.
 --upgrade <pathToExistingApp>
 ```
 
+*NEW IN 43.1.0*
+
 This option will attempt to extract all existing options from the old app, and upgrade it using the current Nativefier CLI.
 
+**IMPORTANT NOTE**
+
+**This action is an in-place upgrade, and will REPLACE the current application. In case this feature does not work as intended or as the user may wish, it is advised to make a backup of your applications before using, or specify al alternate directory as you would when creating a new file.**
```

Comment:
```suggestion
**This action is an in-place upgrade, and will REPLACE the current application. In case this feature does not work as intended or as the user may wish, it is advised to make a backup of the app to be upgraded before using, or specify an alternate directory as you would when creating a new file.**
```